cli/.env.example

```
@@ -1,7 +1,8 @@
 COSMO_API_KEY=cosmo_669b576aaadc10ee1ae81d9193425705
 COSMO_API_URL=http://localhost:3001
 CDN_URL=http://localhost:11000
-PLUGIN_REGISTRY_URL=
+PLUGIN_REGISTRY_URL=host.docker.internal:5050
+PLUGIN_REGISTRY_INSECURE=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PLUGIN_REGISTRY_INSECURE key should go before the PLUGIN_REGISTRY_URL key

(UnorderedKey)